Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -227,8 +222,8 @@ void MaxEnt::exec() {
   MatrixWorkspace_const_sptr inWS = getProperty("InputWorkspace");
   // Number of spectra
   size_t nspec = inWS->getNumberHistograms();
-  // Number of data points
-  size_t npoints = inWS->blocksize() * resolutionFactor;
+  // Number of data points - assumed to be constant between spectra
+  size_t npoints = inWS->y(0).size() * resolutionFactor;
```

Comment:
So what happens now if you call the algorithm with a workspace that does not have constant histogram size?